blockonomics.php

```diff
@@ -396,6 +429,10 @@ public function testSetup()
             if ($response->response_code != 200) {
                 $error_str = $response->data->message;
             }
+            $message = $this->makeWithdraw();
+            if ($message) {
+                $error_str .= $message;
```

Comment:
`makeWithdraw()` can return message even if its not an error. This may cause issues, one of which is wrong message format (error message instead of notification message):

![Screenshot_2019-11-28 Module Manager • DevShop](https://user-images.githubusercontent.com/33265060/69825197-19864880-1217-11ea-9a7b-1069f9e8ecec.png)

Reply:
Fixed error/success messages with commit 306e865